src/content/en/updates/2019/03/prefers-reduced-motion.md

```diff
@@ -0,0 +1,231 @@
+
+```CSS
+/*
+  If the user has expressed their preference for
+  reduced motion, then don't use animations on buttons.
+*/
+@media (prefers-reduced-motion: reduce) {
+  button {
+    animation: none;
+  }
+}
+
+/*
+  If the browser understands the media query and the user
+  explicitly hasn't set a preference, then use animations on buttons.
+*/
+@media (prefers-reduced-motion: no-preference) {
+  button {
+    /* `vibrate` keyframes are defined elsewhere */
+    animation: vibrate 0.3s linear infinite both;
+  }
+}
+```
+
+Note: If you have a lot of animation-related CSS, you can spare your opted-out users from
+downloading it by outsourcing all animation-related CSS into a separate stylesheet that you only
+load conditionally via the `media` attribute on the `link` element&nbsp;😎:<br>
+`<link rel="stylesheet" href="animations.css" media="(prefers-reduced-motion: no-preference)">`
+
+To illustrate how to work with `prefers-reduced-motion` with JavaScript, let's imagine I have
+defined a complex animation with the
+[Web Animations API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API). While
+CSS rules will be dynamically triggered by the browser when the user preference changes, for
+JavaScript animations I have to listen for changes myself, and then manually stop my potentially
+in-flight animations (or restart them if the user lets me):
+
+```JavaScript
+const mediaQuery = window.matchMedia('(prefers-reduced-motion: reduce)');
+mediaQuery.addEventListener('change', () => {
+  console.log(mediaQuery.media, mediaQuery.matches);
+  // Stop JavaScript-based animations.
+});
+```
+
+Note: The parentheses around the actual media query are obligatory:<br>
+`/* 🚫 Wrong */ window.matchMedia('prefers-reduced-motion: reduce')`<br>
+You always have to use this syntax:<br>
+`/* ✅ Correct */ window.matchMedia('(prefers-reduced-motion: reduce)')`
+
+## Demo
+
+I have created a little demo based on Rogério Vicente's amazing
+[🐈 HTTP status cats](https://http.cat/). First, take a moment to appreciate the joke, it's
+hilarious and I'll wait. Now that you're back, let me introduce the
+[demo](https://prefers-reduced-motion.glitch.me). When you scroll down, each HTTP status cat
+alternatingly appears from either the right or the left side. It's a buttery smooth 60fps animation,
+but as outlined above, some users may dislike it or event get motion sick by it, so the demo is
+programmed to respect `prefers-reduced-motion`. This even works dynamically, so users can change
+their preference on-the-fly, no reload required. If a user prefers reduced motion, the non-necessary
+reveal animations are gone, and just the regular scrolling motion is left. The screencast below
+shows the demo in action:
+
+<video width="1098" height="870" intrinsicsize="1098x870" controls>
+  <source src="../../videos/2019/03/prefers-reduced-motion/prefers-reduced-motion.mp4" type="video/mp4">
+</video>
+
+## (Bonus) Forcing reduced motion on all websites
```

Comment:
This section is awesome! But the suggested idea won't work for animations created via the Web Animations API, right? Might be worth noting, or maybe the 'the above hack is not guaranteed to succeed on all websites' text suffices.

Reply:
Right. I have added this as a remark in parentheses.